topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,379 @@
+>
+{: .hands_on}
+
+## Density Functional Theory (DFT) calculation of the Pure Host Material
+
+ This method requires only a single DFT calculation for the host material.  
+ This calculation could be a geometry optimisation, which uses 
+ a self-consistent methodology to estimate the electronic 
+ structure of the material and relax the structure. Pymuon_suite uses the 
+ CASTEP code for running its DFT calculations, and some basic information
+  about how to run a geometry optimisation is available in the [CASTEP documentation](https://castep-docs.github.io/castep-docs/).
+
+Once the relaxed structure and its corresponding electronic structure are obtained, 
+a set of CASTEP output files is generated. We need to load just three of these files
+into Galaxy.  These are the relaxed CASTEP structural file `*-out.cell`, the
+charge density file `*.den_fmt` and the CASTEP output file `*.castep`.
+  
+The CASTEP files for this example have all been pre-generated for this tutorial,
+but in general you would need to create these yourself by running CASTEP simulations
+outside of Galaxy. 
+
+To run the first part of the workflow, on the left menu of the Galaxy web-page
+we look for the `FIND MUON STOPPING SITES` header and first click on `UEP method` and then 
+select the `PyMuonSuite AIRSS UEP Optimise` Galaxy tool. This brings up the central panel of the Galaxy 
+web-page, where can load the CASTEP files; complete the *Generation* parameters to set up the
+creation of randomly populated muonated structures; and complete the *Optimisation* parameters to set up the relaxation the 
+electrostatic forces of the muon in each of the randomly populated muonated structures.
+
+### How to load CASTEP files into Galaxy
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}), from you computer, or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/records/10219558/files/Cu-out.cell 
+>    https://zenodo.org/records/10219558/files/Cu.castep 
+>    https://zenodo.org/records/10219558/files/Cu.den_fmt 
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets to match their file names, if needed.
+> 4. Check that the datatype is correct for each dataset.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+{: .hands_on}
+
+## Create Randomly Populated Muonated Structures
+
+The first step within the Galaxy platform is the population of the 
+host material's interstitial spaces with muons that are 
+randomly distributed muonin accordance with certain physical constraints. 
+
+![Physical Constraints for Random Distribution of Muons](../../images/Create-random-structures.png)
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*

Reply:
Have added the more descriptive alt text to all the images.